support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -106,7 +110,7 @@ export function ThankYouComponent({
 	csrf,
 	payment,
 	productKey = 'Contribution',
-	ratePlanKey,
+	ratePlanKey = 'OneTime',
```

Comment:
I think this default works because one time was always the default in the switch statement below?